static/js/components/Groups.jsx

```diff
@@ -25,6 +25,10 @@ const Groups = () => {
   const roles = useSelector((state) => state.profile.roles);
   const groups = useSelector((state) => state.groups.user);
 
+  if (groups.length === 0) {
```

Comment:
And you can't have the same issue here where groups is not yet loaded, hence does not have length?

Reply:
yes, this one defaults to `[]`.